backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -162,6 +163,15 @@
 ]
 
 
+def has_write_rights(field: str, app_config: CFG, request: TracimRequest):
+    read_only_fields = app_config.USER__READ_ONLY_FIELDS.get(request.candidate_user.auth_type.value)
+    return not (
+        request.current_user.profile.name != "ADMIN"
+        and read_only_fields is not None
+        and field in read_only_fields
+    )
```

Comment:
this logical computation is too difficult to understand.

For something more readable:

if xxx:
    if yyy:
        if zzz:
            return True
return False

Reply:
Will fix this.

Reply:
resolved